lms/djangoapps/bulk_email/migrations/0004_seed_aws_flags.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os
+from django.db import migrations, models
+
+def forwards_func(apps, schema_editor):
+    if os.environ["DJANGO_SETTINGS_MODULE"] == "lms.envs.aws":
+        BulkEmailFlag = apps.get_model("bulk_email", "BulkEmailFlag")
+        db_alias = schema_editor.connection.alias
+        BulkEmailFlag.objects.using(db_alias).create(enabled=True, require_course_email_auth=True)
+
+def reverse_func():
+    pass
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('bulk_email', '0003_config_model_feature_flag'),
+    ]
+
+    operations = [
+        migrations.RunPython(forwards_func, reverse_func),
+    ]
```

Comment:
I'm not sure if we need this data migration or not, but I wanted to throw it out there in case.

The issue: After talking with @nedbat about the Eucalyptus upgrade with regards to this setting -> config model upgrade, we decided it would be best to have no data migration, keep the feature off by default, and tell operators to add a config model on their admin panels to enable this functionality.

However, this will not work smoothly with the main edx.org rollout when this PR lands. If we just go through schema migration -> code rollout, then there will be no values in the table, and the feature (which had previously been turned on via [the settings in common.py](https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L133-L139)) would effectively be turned off until we made our own entry to the admin table (so, an outage of a few minutes).

Is that acceptable? Would this "only do a data migration if `-settings=aws`" approach be better?